resources/views/servers/index.blade.php

```diff
@@ -48,7 +48,7 @@
                                                 @if(!empty(env('PHPMYADMIN_URL')))
                                                     <a href="{{env('PHPMYADMIN_URL' , 'http://localhost')}}" class="dropdown-item text-info"  target="__blank"><i title="manage" class="fas fa-database mr-2"></i><span>Database</span></a>
                                                 @endif
-                                                <form method="post" onsubmit="return submitResult();" action="{{route('servers.destroy' , $server->id)}}">
+                                                <form method="post" onsubmit="return confirmDeletionForm(event, this, 'server');" action="{{route('servers.destroy' , $server->id)}}">
```

Comment:
instead of providing 'server', use '{{__('Server')}}' so it can be translated later